x-pack/plugins/siem/cypress/plugins/index.js

```diff
@@ -37,4 +39,14 @@ module.exports = on => {
     },
   };
   on('file:preprocessor', wp(options));
+
+  on('before:browser:launch', (browser, options) => {
```

Comment:
Since we're stubbing out the downloading functionality, is this still needed?

Reply:
You are right, we can remove it, thx!